sdk/maps/azure-maps-route/README.md

```diff
@@ -78,61 +80,140 @@ route_client = MapsRouteClient(credential=credential)
 
 ## Key concepts
 
+The Azure Maps Route client library for Python allows you to interact with each of the components through the use of a dedicated client object.
+
+### Sync Clients
+
 `MapsRouteClient` is the primary client for developers using the Azure Maps Route client library for Python.
 Once you initialized a `MapsRouteClient` class, you can explore the methods on this client object to understand the different features of the Azure Maps Route service that you can access.
 
+### Async Clients
+
+This library includes a complete async API supported on Python 3.5+. To use it, you must first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/).
+See [azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#transport) for more information.
+
+Async clients and credentials should be closed when they're no longer needed. These
+objects are async context managers and define async `close` methods.
+
 ## Examples
 
 The following sections provide several code snippets covering some of the most common Azure Maps Route tasks, including:
 
-- [Get Route Directions](#get-route-directions)
-- [More Sample Code](#sample-code)
+- [Request and Get Route Directions](#request-and-get-route-directions)
+
```

Comment:
Suggest to remove empty line between list of items (line 103 and line 105)

Reply:
removed.